gnucash/python/init.py

```diff
@@ -96,17 +97,20 @@ def quit (self):
 # Change this to "if True:" to switch on a python console at gnucash
 # startup:
 # shelltype can either be "python" or "ipython" (the latter is not yet fully functional)
-if False:
-    shelltype = "python"
+if True:
+    shelltype = "ipython"
     title = "gnucash "+shelltype+" shell"
     banner_style = 'title'
     banner = "Welcome to "+title+"!\n"
-    console = Console(argv = [], shelltype = shelltype, banner = [[banner, banner_style]], size = 100)
-
+   
     window = Gtk.Window(type = Gtk.WindowType.TOPLEVEL)
     window.set_position(Gtk.WindowPosition.CENTER)
     window.set_default_size(800,600)
     window.set_border_width(0)
+    
+    console = Console(argv = [], shelltype = shelltype, banner = [[banner, banner_style]],
```

Comment:
Moving initialisation of console down makes it possible to access `window` in shell as well. Console itself can be found by `window.get_children()[0]`.